build.gradle.kts

```diff
@@ -11,7 +11,8 @@ import java.net.URI
 plugins {
     java
     id("org.springframework.boot") version "3.4.0"
-    id("org.jlleitschuh.gradle.ktlint") version "12.1.2"
+    id("org.jetbrains.kotlin.plugin.spring") version "2.1.0"
+    //id("org.jlleitschuh.gradle.ktlint") version "12.1.2"
```

Comment:
If it's not needed - can we remove it?

Reply:
I would like to keep it and discuss it with @pschwager and @patwlan how we can tune the config or how I can reformat the code.
Currently I think the config is not good.